docs/make.jl

```diff
@@ -8,23 +8,52 @@ DocMeta.setdocmeta!(HOHQMesh,     :DocTestSetup, :(using HOHQMesh);     recursiv
 # Get Trixi root directory
 hohqmesh_root_dir = dirname(@__DIR__)
 
-# Copy list of authors to not need to synchronize it manually
-authors_text = read(joinpath(hohqmesh_root_dir, "AUTHORS.md"), String)
+# Copy list of authors to not need to synchronize it manually.
+# Since the authors header exists twice we create a unique identifier for the docs section.
+authors_text = read(joinpath(dirname(@__DIR__), "AUTHORS.md"), String)
+authors_text = replace(authors_text,
+                       "# Authors" => "# [Authors](@id hohqmesh_authors)")
 write(joinpath(@__DIR__, "src", "authors.md"), authors_text)
 
+# Copy code of conduct to not need to synchronize it manually
+code_of_conduct_text = read(joinpath(dirname(@__DIR__), "CODE_OF_CONDUCT.md"), String)
+code_of_conduct_text = replace(code_of_conduct_text,
+                               "[AUTHORS.md](AUTHORS.md)" => "[Authors](@ref hohqmesh_authors)")
+write(joinpath(@__DIR__, "src", "code_of_conduct.md"), code_of_conduct_text)
```

Comment:
Don't forget to add this file (and all others that are created in `make.jl`) to an appropriate `.gitignore` file